go/interpreter/evmc/evmc_interpreter.go

```diff
@@ -166,7 +166,13 @@ type hostContext struct {
 }
 
 func (ctx *hostContext) AccountExists(addr evmc.Address) bool {
-	return ctx.context.AccountExists(tosca.Address(addr))
+	// Although the EVMC function name asks for the existence of an account,
+	// it is actually referring to the emptiness of an account. The concept
+	// of an existing or non-existing account is a DB concept that is not
+	// exposed to any interpreter implementation.
```

Comment:
is this confusion something we could fix in the names of the interface's functions as well ?

Reply:
This interface is defined by an external dependency -- the [here](https://github.com/ethereum/evmc/blob/5a66578af363b28698835aa45eb920f20ae79f56/bindings/go/evmc/host.go#L89), in the EVMC library.